hack/make-rules/verify.sh

```diff
@@ -35,7 +35,6 @@ EXCLUDED_PATTERNS=(
   "verify-linkcheck.sh"          # runs in separate Jenkins job once per day due to high network usage
   "verify-*-dockerized.sh"       # Don't run any scripts that intended to be run dockerized
   "verify-govet-levee.sh"        # Do not run levee analysis by default while KEP-1933 implementation is in alpha.
-  "verify-golangci-lint.sh"             # TODO(dims) Need to get this running with golang 1.18
```

Comment:
I don't understand why we are removing this?

Reply:
this was temporarily excluded last night to get CI working again on go1.18, and this is removing the exclusion

Reply:
ACK, missed which list it was in